cspell.json

```diff
@@ -0,0 +1,15 @@
+{
+    "version": "0.2",
+    "language": "en",
+    "words": [
+        "CNCF",
+        "OTLP",
+        "OpenTelemetry",
+        "OTel",
+        "daemonset",
+        "zipkin",
+        "protobuf",
+        "parentbased",
+        "tabpane"
+    ]
+}
```

Comment:
We already have a cSpell file under https://github.com/open-telemetry/opentelemetry.io/blob/main/.vscode/cspell.json. I'd suggest making additions there.

Reply:
missed that! Will fix it.